crates/sui-move-build/src/lib.rs

```diff
@@ -689,19 +713,28 @@ impl CompiledPackage {
     }
 }
 
-impl Default for BuildConfig {
```

Comment:
Note to reviewers: I removed the `Default` implementation since there is arguably no reasonable default.